index.js

```diff
@@ -64,229 +63,234 @@ var defaults = {
   preserveAtRulesOrder: false
 };
 
-module.exports = postcss.plugin("postcss-css-variables", function(options) {
+module.exports = (options = {}) => {
```

Comment:
Does supporting this new syntax drop support for older versions of PostCSS? Which versions?

Reply:
It looks like this is a hard break. I tried a minimal build task with postcss@^7 and this branch and got this error:

> Error: PostCSS plugin postcss-css-variables requires PostCSS 8.
> Migration guide for end-users:
> https://github.com/postcss/postcss/wiki/PostCSS-8-for-end-users